package.json

```diff
@@ -7,6 +7,7 @@
     "prepare": "husky install",
     "test": "jest --verbose",
     "test-e2e": "playwright test",
+    "docker-test-e2e": "docker run --rm --ipc=host -v $PWD:/usr/src/app -w /usr/src/app mcr.microsoft.com/playwright:v1.40.1-jammy /bin/bash -c 'apt update && apt install --yes make gcc g++ && npm run clean && npm install && npm run test-e2e'",
```

Comment:
The update and install of make, gcc and g++ can likely be sped up by using a `Dockerfile` to cache the resultant image. Please consider giving that a try.

Reply:
Potentially even the required `npm install` of Linux dependencies can be layered in via `Dockerfile`.

Reply:
Concerns addressed in subsequent commits.